pkg/engine/loadbalancers.go

```diff
@@ -140,73 +108,115 @@ func CreateLoadBalancer(prop *api.Properties, isVMSS bool) LoadBalancerARM {
 		},
 	}
 
-	if prop.OrchestratorProfile.KubernetesConfig.LoadBalancerSku == api.StandardLoadBalancerSku {
-		udpRule := network.LoadBalancingRule{
-			Name: to.StringPtr("LBRuleUDP"),
-			LoadBalancingRulePropertiesFormat: &network.LoadBalancingRulePropertiesFormat{
-				FrontendIPConfiguration: &network.SubResource{
-					ID: to.StringPtr("[variables('masterLbIPConfigID')]"),
-				},
-				BackendAddressPool: &network.SubResource{
-					ID: to.StringPtr("[concat(variables('masterLbID'), '/backendAddressPools/', variables('masterLbBackendPoolName'))]"),
-				},
-				Protocol:             network.TransportProtocolUDP,
-				FrontendPort:         to.Int32Ptr(1123),
-				BackendPort:          to.Int32Ptr(1123),
-				EnableFloatingIP:     to.BoolPtr(false),
-				IdleTimeoutInMinutes: to.Int32Ptr(5),
-				LoadDistribution:     network.Default,
-				Probe: &network.SubResource{
-					ID: to.StringPtr("[concat(variables('masterLbID'),'/probes/tcpHTTPSProbe')]"),
-				},
-			},
-		}
-		*loadBalancer.LoadBalancer.LoadBalancerPropertiesFormat.LoadBalancingRules = append(*loadBalancer.LoadBalancer.LoadBalancerPropertiesFormat.LoadBalancingRules, udpRule)
-	}
-
-	if isVMSS {
-		inboundNATPools := []network.InboundNatPool{
+	if !prop.OrchestratorProfile.IsPrivateCluster() {
```

Comment:
Because we are now re-using this master LB for private cluster + Standard LB scenarios (in order to grant outbound internet access to the master(s)) we remove the LB attributes that don't make sense in a private cluster scenario, e.g.,, inbound TCP 443 and 22 access.